.circleci/config.yml

```diff
@@ -83,6 +83,7 @@ jobs:
             - run: pip install .[sklearn,tf-cpu,torch,testing,sentencepiece,torch-speech,vision]
             - run: pip install torch-scatter -f https://pytorch-geometric.com/whl/torch-1.10.0+cpu.html
             - run: pip install tensorflow_probability
+            - run: pip install https://github.com/kpu/kenlm/archive/master.zip
```

Comment:
What is this package? The import error indicate to do `pip install pyctcdecode` later on and do not give any instruction to install this.

Reply:
`pyctcdecode` optionally depends on kenlm if the user would like to use a `kenlm` language model. In the future, there will probably be more language models that don't require `kenlm`. 

So IMO, it's the responsibility of the `pyctcdecode` package to throw a good error in case a user requests `pyctcdecode` with a `kenlm` language model. However since at the moment the only language model support is based on `kenlm` I can also throw a nice error message on our side.

Reply:
Is there a plan to add a realy Python package for `kenlm`? This is a bit heavy :-(

Reply:
Hmm, I'm really not sure the `kenlm` repo doesn't seem to be super active: https://github.com/kpu/kenlm .

It is however the by far most library for language model supported ASR. 
Flashlight uses it: https://github.com/flashlight/flashlight/tree/main/bindings/python#dependencies among many other libraries.

Reply:
Espnet uses it as well: https://github.com/espnet/espnet/blob/master/espnet/nets/scorers/ngram.py